src/components/RecoverPassword.tsx

```diff
@@ -0,0 +1,24 @@
+import RecoverPassowrdForm from "components/recoverPassword/Form";
```

Comment:
There's a typo here. And obviously where it is implemented.
```suggestion
import RecoverPasswordForm from "components/recoverPassword/Form";
```